### ISSUE TITLE
Support parameters.json

### ISSUE THREAD

**Issue by berland** [equinor/fmu-ensemble#66]

**Comment by anders-kiaer** [comment]
I guess `parameters.json` is the «future», and `parameters.txt` eventually will be removed. But are the values conflicting? Or only the keys in the dictionary?

Should `ecl2df` know about `parameters.json` (which is a fmu/ert concept) going forward? Conceptually I like the idea that `ecl2df` is a high level Eclipse file handler, and then fmu-ensemble’s Mail role is to link realization number/iteration number/parameters.json/other FMU metadata, and call e.g. `ecl2df` where appropriate.

**Comment by berland** [comment]
Good point, `ecl2df` should probably import this feature from `fmu-ensemble` for the sake of the `ecl2csv smry -p` command.
